### PR TITLE
Fix: Malformed prop-value, prevents editor from loading [ED-22035]

### PR DESCRIPTION
**Pull request by Batsirai-muchareva**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### ✨ PR Description
Purpose: Fix malformed size variable prop-values that prevented editor from loading by adding validation and default unit handling in Prop_Type_Adapter.

Main changes:
- Added null-safety checks and default unit fallback in from_storage() for malformed size variables
- Moved schema lookup inside closure to fix undefined variable error in to_storage() method
- Added comprehensive test coverage for edge cases with empty, null, and malformed variable values

_Generated by LinearB AI and added by gitStream._
<sub>AI-generated content may contain inaccuracies. Please verify before using.
💡 **Tip:** You can customize your AI Description using **Guidelines** [Learn how](https://docs.gitstream.cm/automation-actions/#describe-changes)</sub>

### CODE REVIEW

File: tests/phpunit/elementor/modules/variables/adapters/test-prop-type-adapter.php [hunk 1]
```diff
@@ -556,4 +557,226 @@ public function test_to_storage__handles_percentage_values() {
...
+				'e-gv-null-value' => [
+					'type' => Size_Variable_Prop_Type::get_key(),
+					'label' => 'Size String',
+					'value' => null,
+				],
+				'e-gv-size-empty' => [
+					'type' => Size_Variable_Prop_Type::get_key(),
+					'label' => 'Size Empty',
+					'value' => [
+						'$$type' => 'size',
```
Comment (mike-elementor) [inline tests/phpunit/elementor/modules/variables/adapters/test-prop-type-adapter.php:706]: what if `$$type` is missing?
what if `value` is missing?

File: tests/phpunit/elementor/modules/variables/adapters/test-prop-type-adapter.php [hunk 2]
```diff
@@ -556,4 +557,385 @@ public function test_to_storage__handles_percentage_values() {
...
+					'type' => Size_Variable_Prop_Type::get_key(),
+					'label' => 'Size Null',
+					'value' => [
+						'$$type' => 'size',
+						'value' => [
+							'size' => null,
+							'unit' => null,
+						],
+					],
+				],
```
Comment (mike-elementor) [inline tests/phpunit/elementor/modules/variables/adapters/test-prop-type-adapter.php:723]:
```suggestion
				'e-gv-size-null' => [
					'type' => Size_Variable_Prop_Type::get_key(),
					'label' => 'Size Null',
					'value' => [
						'$$type' => 'size',
						'value' => [
							'size' => null,
							'unit' => null,
						],
					],
				],
				'e-gv-size-malformed' => [
					'type' => Size_Variable_Prop_Type::get_key(),
					'label' => 'Size Malformed',
					'value' => [
						'$$type' => 'size',
						'value' => [],
					],
				],
```

**Review by mike-elementor — APPROVED** [review]
